marge/app.py

```diff
@@ -27,138 +28,153 @@ def time_interval(s):
         raise argparse.ArgumentTypeError('Invalid time interval (e.g. 12[s|min|h]): %s', s)
 
 
-def _parse_args(args):
-    parser = argparse.ArgumentParser(description=__doc__)
-
-    def arg(name, env_var=True, **kwargs):
-        if env_var:
-            env_var_name = 'MARGE_' + name[2:].replace('-', '_').upper()
-            env_var_val = os.getenv(env_var_name)
-            if env_var_val:
-                if kwargs.get('action') in ['store_true', 'store_false']:
-                    env_var_val = env_var_val.lower() in ['true', 'yes', '1']
-                kwargs = dict(kwargs, default=env_var_val, required=False)
+def _parse_config(args):
 
-        parser.add_argument(name, **kwargs)
+    for bad_arg in ['--auth-token', '--ssh-key']:
+        if bad_arg in args:
+            raise Exception('"%s" can only be set via ENV var or config file.' % bad_arg)
```

Comment:
@mrusu91 I can still call marge with `--auth-token=everybody-can-see-my-token` and this test won't catch it, right?

Reply:
Also, can we maybe add a test for this case in `test_app.py`?

Reply:
hmm, that's true, will find  a solution and write a test